docs/configure_your_operations.md

```diff
@@ -386,3 +386,7 @@ class Book implements ResourceInterface
 | app_admin_book_bulk_delete | DELETE          | /admin/books/bulk_delete |    
 | app_shop_book_index        | GET             | /books/                  |
 | app_shop_book_show         | GET             | /books/{id}              |
+
+**[Go back to the documentation's index](index.md)**
+
+**[> Next chapter: Resource factories](resource_factories.md)**
```

Comment:
I suppose it should be `redirect.md` (see [here](https://github.com/Sylius/SyliusResourceBundle/pull/682/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R25))?